lib/config/options/index.ts

```diff
@@ -2394,6 +2403,16 @@ const options: RenovateOptions[] = [
     env: false,
     advancedUse: true,
   },
+  {
+    name: 'header',
+    description:
+      'An object that includes fields to be forwarded to the HTTP request header.',
```

Comment:
I don't understand this sentence, it feels too much like a soup of nouns:

- an object
- fields
- HTTP request header

And we have some weak verbs:

- includes
- forwarded
- 
I'd like the `description` to explain how to use it, and when to use it. I'm thinking of something like this:

> Put fields to be forwarded to the HTTP request header in the `header` config option.